bank-sdk/sdk/src/main/res/layout-sw600dp-land/gbs_fragment_digital_invoice.xml

```diff
@@ -172,4 +172,13 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent" />
 
+    <androidx.compose.ui.platform.ComposeView
+        android:id="@+id/compose_view"
```

Comment:
[Request] Please add gbs prefix to these ids (there are 4 xml files with this id).

Reply:
Done